src/App.vue

```diff
@@ -525,11 +526,11 @@ export default {
 
       const draft = await LegalServices.fetchDraftApp(this.tempRegNumber)
 
-      // Handle Draft filings
+      // handle draft appliation
```

Comment:
Just a typo here on "application", otherwise looks good

Reply:
Well spotted @leodube-aot for a Friday afternoon :D

Reply:
Fxed. I cnt wt to go hm.